### ISSUE TITLE
Seeing item appear out of order

### ISSUE THREAD

**Issue by bentsai** [esturcke/hog#2]
```
  6.0 GB Brave Browser Beta
  1.7 GB WindowServer
  0.0 GB Webex
707.4 MB Discord
693.4 MB Slack
```

**Comment by esturcke** [comment]
It think this is a bug with number rounding. I think that should be 1.0 GB. I updated dependencies. Maybe this is fixed in 0.3.

**Comment by esturcke** [comment]
I have not seen this in a while. I think it's fixed now.
